packages/runtime/src/dataViews/consumption/DecidableRequestItemDVOs.ts

```diff
@@ -20,6 +20,12 @@ export interface DecidableCreateAttributeRequestItemDVO extends DecidableRequest
     attribute: DraftIdentityAttributeDVO | DraftRelationshipAttributeDVO;
 }
 
+export interface DecidableDeleteAttributeRequestItemDVO extends DecidableRequestItemDVO {
+    type: "DecidableDeleteAttributeRequestItemDVO";
+    attributeId: string;
+    attribute: LocalAttributeDVO;
```

Comment:
The other DecidableRequestItemDVOs (for example the DecidableShareAttributeRequestItemDVO) use DraftIdentityAttributeDVO and/or DraftRelationshipAttributeDVO at this point.